database/postgres-cm.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: postgres-secret
+  labels:
+    app: postgres
+data:
+  POSTGRES_DB: <db_name>    # postgresdb
+  POSTGRES_USER: <db_user>    # admin or ps_user
+  POSTGRES_PASSWORD: <db_pass>    # a secure password
```

Comment:
I think `POSTGRES_PASSWORD` should be put in a Secret instead of a ConfigMap

Reply:
Chris mentioned this before. I thought that this was the faster way to do it. I was check it how to change and it was an easy change. So I remove the ConfigMap and added a Secret.